bundle-chroot-builder.py

```diff
@@ -151,6 +156,37 @@ def clean_bundle(out_dir, bundle, bundles, yum_cmd):
     os.chdir(prev_dir)
 
 
+def write_default_server_ini(state_dir, server_conf):
+    """
+    Write default server.ini file in the state dir in the following format
+
+    [Server]
+    emptydir={state_dir}/empty/
+    imagebase={state_dir}/image/
+    outputdir={state_dir}/www/
+
+    [Debuginfo]
+    banned=true
+    lib=/usr/lib/debuginfo/
+    src=/usr/src/debuginfo/
```

Comment:
The docstring might get stale over time, but I would fix up the two lines above as well.

Reply:
how so? This is how they will appear in server.ini

Reply:
s|debuginfo/|debug/| for those two lines

Reply:
oh, oops